src/features/gdpr.py

```diff
@@ -1,17 +1,18 @@
 import re
 
 from app.models import Explanation, StarCase
-from features.metadata_base import MetadataBase
-from features.website_manager import WebsiteData
-from lib.constants import STRICT_TRANSPORT_SECURITY, VALUES
+from core.metadata_base import MetadataBase
+from core.website_manager import WebsiteData
+from lib.constants import STRICT_TRANSPORT_SECURITY
 
 
+@MetadataBase.register(key="g_d_p_r")  # fixme use GDPR as key!
 class GDPR(MetadataBase):
     tag_list = ["impressum"]
     decision_threshold = 0.3
 
     @staticmethod
-    def _check_https_in_url(website_data: WebsiteData) -> list:
+    def _check_https_in_url(website_data: WebsiteData) -> list[str]:
```

Comment:
Since I see this a lot, how about using a constant for this type? Just a thought, i.e., optional